pkg/utils/validation/cidr/disjoint_test.go

```diff
@@ -392,6 +416,30 @@ var _ = Describe("utils", func() {
 			})),
 			))
 		})
+
+		It("should fail due to seed pod network and shoot node network overlap", func() {
+			var (
+				podsCIDR     = "2001:0db8:35a3::/113"
+				servicesCIDR = "2001:0db8:45a3::/113"
+				nodesCIDR    = seedPodsCIDRIPv6
+			)
+
+			errorList := ValidateNetworkDisjointedness(
+				field.NewPath(""),
+				&nodesCIDR,
+				&podsCIDR,
+				&servicesCIDR,
+				&seedNodesCIDRIPv6,
+				seedPodsCIDRIPv6,
+				seedServicesCIDRIPv6,
+				false,
+			)
+
+			Expect(errorList).To(ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{
+				"Type":  Equal(field.ErrorTypeInvalid),
+				"Field": Equal("[].nodes"),
```

Comment:
Can you also include the "Detail" of the error?

Reply:
Addressed with https://github.com/gardener/gardener/pull/8353/commits/2b7c4df95fdd01469ddcccddd8e9cb17cb387eea.

However, no other test in this file actually checks `Detail`. I only adapted the new tests for now.